ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -6,18 +6,62 @@
 //
 
 import UIKit
+import Kingfisher
+import ProgressHUD
 
-class ImagesListViewController: UIViewController {
-
+final class ImagesListViewController: UIViewController {
     @IBOutlet private var tableView: UITableView!
+    private let imagesListService = ImagesListService.shared
     private let showSingleImageSegueIdentifier = "ShowSingleImage"
-    private let photosName: [String] = Array(0...19).map { "\($0)" }
+    private var photos: [Photo] = []
+    private var ImagesListObserver: NSObjectProtocol?
+    private let today = Date()
     private lazy var dateFormatter: DateFormatter = {
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "d MMMM yyyy"
+        dateFormatter.locale = Locale(identifier: "ru_RU")
         return dateFormatter
     } ()
     
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.contentInset = UIEdgeInsets(top: 12, left: 0, bottom: 12, right: 0)
+        
+        imagesListService.fetchPhotosNextPage()
+        
+        ImagesListObserver = NotificationCenter.default.addObserver(
+            forName: ImagesListService.didChangeNotification,
+            object: nil,
+            queue: .main) { [weak self] _ in
+                guard let self else { return }
+                updateTableViewAnimated()
+                return
+            }
+    }
```

Comment:
```suggestion
        ImagesListObserver = NotificationCenter.default.addObserver(
            forName: ImagesListService.didChangeNotification,
            object: nil,
            queue: .main
        ) { [weak self] _ in
            guard let self else { return }
            updateTableViewAnimated()
        }
    }
```

Если последнюю скобку перенести на следующую строку, то будет лучше по чтению по файлу